copter/source/docs/traditional-helicopter-autorotation-mode.rst

```diff
@@ -0,0 +1,170 @@
+collective angles being applied, resulting in blade stall or even rotor stop.
+
+When initiated, the bail-out phase will read the requested climb or descent rate from the operators throttle stick position.  The aircraft will then be decelerated 
+over a set period of time (see parameter AROT_BAIL_TIME).  At the end of which, the requested vertical speed will match the magnitude requested on the stick.  
+The time spent in this phase is typically of the order of two seconds.  Smaller helicopters (550 size and below) may require this time to be extended to three or 
+four seconds.  When deciding/tuning, it is conservative to opt for a longer time and initiate a practice bail-out with a large height margin for recovery.
+
+.. note::
+
+    The motor ramp time defined by H_RSC_RAMP_TIME is ignored during bail-out, and a hard-coded ramp time of one second is used.
+
+
+.. warning::
+
+    The ramp time defined by H_RSC_RAMP_TIME is now only observed when the aircraft is on ground and the "landed flag" is set true.  It often takes ArduCopter a second 
+    to register that the aircraft has landed once on the ground.  Therefore, if the interlock is re-engaged immediately after the aircraft touches down the one second 
+    ramp time of the bail-out phase can be initiated on the ground.  Caution is therefore required when running autorotations in quick succession to prevent rapid rotor 
+    spool up on ground.
+
+
+Mode Setup
+==========
+
+This flight mode is currently under development.  It is available in Copter 4.0 as SITL only, for those that want to test the entry, glide, and bail-out phases.  
+For the most up-to-date information please see this `Discourse Topic <https://discuss.ardupilot.org/t/autonomous-autorotation-gsoc-project-blog/42139>`__ .  
+Once the mode has been sufficiently tested the final instructions will be published and maintained on this wiki page.
+
+
+
+Mode Entry
+==========
+
+The autonomous autorotation flight mode cannot be entered via a mode switch, as is done with other flight modes.  In an attempt to mimic the autorotation procedure 
+followed when flying a manual autorotation, the mode is entered and exited using the motor interlock switch.  The autonomous flight mode can only be entered if 
+the interlock is disengaged whilst flying in a non-manual collective mode.  When exiting the autorotation the flight mode is returned to the mode from which it came 
+(e.g. Altitude Hold -> Autonomous Autorotation -> Altitude Hold).  The only exception to this is when the aircraft enters the autorotation from an auto mission.  
+In which case, the mode is exited into Altitude Hold.  This is done to prevent the aircraft from potentially resuming its auto mission and flying further away 
+from home, whereby in the scenario it has autonomously recovered from a power failure and is therefore high risk for another power failure.
+
+The diagram below illustrates the mode switchology:
```

Comment:
learned a new word today!....perhaps "mode switching scheme" would be more straightforward than switchology?  your choice

Reply:
Bill and I discussed this at length a while back.  I wasn't sure if it was a real word at the time but it is the correct technical term to use.  I think it is straight forward enough to infer its meaning as well.